### PR TITLE
chore(code-coverage): Start getting code coverage artifacts saving

### PR DESCRIPTION
**Pull request by cwoolum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 [![Build Status][circle-badge]][circle-badge-url]
 [![Code Style: Prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg)](https://github.com/prettier/prettier)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+[![Coverage Status](https://coveralls.io/repos/github/ionic-team/v4-migration-tslint/badge.svg?branch=feature%2Fadd-code-coverage)](https://coveralls.io/github/ionic-team/v4-migration-tslint?branch=feature%2Fadd-code-coverage)
```
Comment (dwieeb) [inline README.md]: branch needs updating now or later

File: package.json [hunk 2]
```diff
@@ -59,12 +64,13 @@
...
   },
   "release": {
-    "publish": [
-      {
-        "path": "@semantic-release/npm",
-        "npmPublish": true,
-        "pkgRoot": "./dist"
-      }
-    ]
+    "publish": [{
```
[the remark this answers is not in the document]
Comment (cwoolum) [reply package.json]: I'm on NPM 6

File: README.md (hunk 1 above)
Comment (cwoolum) [reply README.md]: fixed

**Pull request merged**